Gruntfile.js

```diff
@@ -244,7 +244,7 @@ module.exports = function(grunt) {
           {
             expand: true,
             cwd: 'ddocs/',
-            src: '**/*',
+            src: ['**/*', '!**/translations/*'],
```

Comment:
That's something I wasn't 100% on where we wanted to store them so built a script to merge. I am excluding the translations from being copied here because the script is merging and outputting. Standard translations overwrite anything in the default translations but standard config translations didn't have all the translations. Having them in one file would get rid of excess code and probably be less confusion. I can generate the file and move them to standard config translations and then we would just load that file from now on.

Reply:
That'd be ideal. It's even more complicated because we have a tool called POEditor which imports translations from the webapp repo and provides a handy UI for translators to fill in each language and then export the translations back into the repo.

This will need to be reconfigured but we can sort that after the merge.